docs/getting-started.stories.mdx

```diff
@@ -175,48 +176,19 @@ import ReactDOM from "react-dom";
 
 ### Theming and Design Tokens
 
-Currently we are implementing Design Tokens into our components. Implementing Design Tokens implies migration from ThemeProvider to CSS Variables. Because of this we temporary support two theming systems.
+Currently, we are implementing Design Tokens into our components. Implementing Design Tokens implies migration from styled-component's ThemeProvider to CSS variables. Because of this, we temporarily are supporting two themes system .
```

Comment:
```suggestion
We are currently implementing Design Tokens into our components. Implementing Design Tokens implies migration from styled-component's ThemeProvider to CSS variables. Because of this, we are temporarily supporting both theming systems.
```